### DISCUSSION TITLE
How do I remove Cookie and Accept item in request header?

### DISCUSSION THREAD

**Discussion by ghost** [hoppscotch/hoppscotch#3564] [Q&A]
Postman does not include it when the cookie is empty, but Hoppscotch automatically adds an empty string. How can I remove it?
For example:
When I sending a empty request to echo.hoppscotch.io.

Hoppscotch get it:
```json
{
  "method": "GET",
  "args": {},
  "data": "",
  "headers": {
    "accept": "*/*,image/webp",
    "cdn-loop": "netlify",
    "cookie": "",
    "host": "echo.hoppscotch.io"
    // I manually deleted x-country, x-nf-site-id etc and it did nothing in this question.
  },
  "path": "/",
  "isBase64Encoded": true
}
```

But postman get:
```json
{
  "method": "GET",
  "args": {},
  "data": "",
  "headers": {
    "cdn-loop": "netlify",
    "host": "echo.hoppscotch.io"
  },
  "path": "/",
  "isBase64Encoded": true
}
```

**Comment by liyasthomas** [comment]
@AndrewBastin shall I move this to issues?

**Comment by AndrewBastin** [comment]
@ikspress This is a browser limitation and something which is out of our control on the web app side of things. On the desktop app, this is something we can consider though.

What is the use case though ? Is there any reason why you do not want to send the "Accept" or the "Cookie" header.

**Reply by ghost** [reply]
Oh, hello. I noticed this when I get first started. At first, I thought it was just a hidden item. I am using the desktop app, could you implement this feature?

**Comment by bvwidt** [comment]
Is this still considered to be implemented?
I have a concrete use case where a web server does behave differently if a cookie is sent wether or not it is empty or not.
Unfortunately I cannot edit the web server itself and that is why I would like the empty Cookie header to be removed.
